Tasks/git-clone-accesstoken/task.yaml

```diff
@@ -0,0 +1,48 @@
+# This is a git repository cloning task for Dev Box
+
+$schema: 1.0
+name: git-clone-accesstoken
+description: Clones a git repository with access token.
+author: Microsoft Corporation
+command: ./main.ps1 -RepositoryUrl {{ repositoryUrl }} -Directory {{ directory }} -Branch {{ branch }} -At {{ at }}
+parameters:
+  repositoryUrl:
+    default: ''
+    type: string
+    required: true
+    description: The URL of the repository to clone.
+  at:
+    default: ''
+    type: string
+    required: true
+    description: The access token to clone the repository.
+  directory:
+    default: C:\Workspaces
+    type: string
+    required: false
+    description: |
+      The directory under which the repository will be cloned. This is the
+      equivalent of the working directory under which the git command will
+      execute.
+  branch:
+    default: ''
+    type: string
+    required: false
+    description: The branch to clone.
+documentation:
+  notes: |
+    This task allows you to clone a repository with access token.
+  examples:
+    - name: git-clone
```

Comment:
please change the name to git-clone-accesstoken

Reply:
Thanks for catching this. It's fixed.